pkg/systemd/services.html

```diff
@@ -92,42 +92,30 @@
       <div class="panel-heading">{{Unit.Description}}</div>
       <div class="list-group">
         <div class="list-group-item">
-          <table>
-            <tr>
-              <td>
-                <span>{{Unit.ActiveState}}</span>
-                ({{Unit.SubState}})
-                <br/>
-                {{Since}}
-              </td>
-              <td>
-                {{{UnitButton}}}
-              </td>
-            </tr>
+          {{#HasLoadError}}
+            <div class="alert alert-danger">
+              <span class="pficon pficon-error-circle-o"></span>
+              <strong>{{Unit.LoadState}}</strong> {{LoadError}}
+            </div>
+          {{/HasLoadError}}
+          <table class="form-table-ct">
+            <tr><td class="top-right" translate>Status</td><td>{{Unit.ActiveState}} ({{Unit.LoadState}}) {{{UnitButton}}}</td></tr>
+            {{#Since}}
+              <tr><td class="top-right" translate>Running Since</td><td>{{Since}}</td></tr>
+            {{/Since}}
+            <tr><td class="top-right" translate>Automatic Startup</td><td>{{Unit.UnitFileState}} {{{FileButton}}}</td></tr>
```

Comment:
If there is a `LoadError`, `UnitFileState` and `FragmentPath` might be empty.  We might want to show some special value in this case, like "n/a" or "-".

Reply:
Ah, your screenshot shows exactly this case.

How can we even have services in the list that are not found?  They appear when another service has a dependency on some service that doesn't actually exists.  So we could filter them out in the big list, and disable links to them (with a tooltip that says "Not found").

This leaves only the case when people manually modify the URL, which we need to handle but it can look different from other load errors of legitimate units.

Reply:
See #10849

Reply:
#10849 got merged which means that this concern here does not apply any longer.  Showing FragmentPath also for a load error is actually helpful.